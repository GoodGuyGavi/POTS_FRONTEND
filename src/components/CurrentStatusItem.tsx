import React from "react";
import { Card, Feed } from "semantic-ui-react";
import CurrentStatus from "./CurrentStatus";

export interface ICurrentStatusitemProps {
  status?: any;
}

const CurrentStatusItem = (props: ICurrentStatusitemProps) => {
  return (
    <Card style={{ maxHeight: 370, overflow: "auto" }}>
      <CurrentStatus />
      {props.status.map((current: any) => (
        <Card.Content style={{ marginLeft: 10 }}>
          <Feed>
            <Feed.Event>
              <Feed.Content>
                <Feed.Summary>{current.currentstatus}</Feed.Summary>
              </Feed.Content>
            </Feed.Event>
          </Feed>
        </Card.Content>
      ))}
    </Card>
  );
};

export default CurrentStatusItem;
