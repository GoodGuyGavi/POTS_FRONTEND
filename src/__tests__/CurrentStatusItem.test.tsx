/**
 * @format
 */

import React from "react";
import { shallow } from "enzyme";
// import ShallowRenderer from "react-test-renderer/shallow";

// // Note: test renderer must be required after react-native.
// import renderer from "react-test-renderer";

import CurrentStatusItem from "../components/CurrentStatusItem";

const props = {
  title: "Title"
};

describe("Current Status Item", () => {
  test("renders correctly", () => {
    expect(shallow(<CurrentStatusItem {...props} />)).toMatchSnapshot();
  });
});
