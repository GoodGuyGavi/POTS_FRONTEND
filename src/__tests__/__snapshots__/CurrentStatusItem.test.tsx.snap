// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Current Status Item renders correctly 1`] = `ShallowWrapper {}`;
